src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/caseload/ApproverCaseloadService.kt

```diff
@@ -0,0 +1,26 @@
+package uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.caseload
+
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import org.springframework.stereotype.Service
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.model.LicenceSummaryApproverView
+import uk.gov.justice.digital.hmpps.createandvaryalicenceapi.service.PrisonApproverService
+
+@Service
+class ApproverCaseloadService(
+  private val prisonApproverService: PrisonApproverService,
```

Comment:
I split out the `PrisonApproverService` recently so that there would be a place for the caseload logic.

Reply:
we've had a bit of a proliferation of services.. maybe we could keep them separate and look to merge later

We want to avoid having multiple layers of transformation but to pull back the changes from the frontend by porting it, keeping it as similar as possible I guess it makes sense to keep the two layers

Reply:
Think maybe we need a bit of a modelling session in the next few weeks to try to instill some structure in to the backend